forum/serializers/comments_api.py

```diff
@@ -0,0 +1,89 @@
+"""
+Serializer for the comment data.
+"""
+
+from typing import Any
+
+from rest_framework import serializers
+
+from forum.serializers.contents import EditHistorySerializer
+from forum.serializers.votes import VoteSummarySerializer
+
+
+class EndorsementSerializer(serializers.Serializer[dict[str, Any]]):
+    """
+    Serializer for handling endorsement of a comment
+
+    Attributes:
+        user_id (str): The endorsement user id.
+        time (datetime): The timestamp of when the user has endorsed the comment.
+    """
+
+    user_id = serializers.CharField()
+    time = serializers.DateTimeField(format="%Y-%m-%dT%H:%M:%SZ")
+
+    def create(self, validated_data: dict[str, Any]) -> Any:
+        """Raise NotImplementedError"""
+        raise NotImplementedError
+
+    def update(self, instance: Any, validated_data: dict[str, Any]) -> Any:
+        """Raise NotImplementedError"""
+        raise NotImplementedError
+
+
+class CommentsAPISerializer(serializers.Serializer[dict[str, Any]]):
```

Comment:
can we reuse the `UserCommentSerializer` class  and make adjustments over there instead of making a new serializer class?